trakt_scrobbler/trakt_interface.py

```diff
@@ -40,6 +40,12 @@ def get_trakt_id(title, item_type, year=None):
 
     logger.debug(f'Searching trakt: Title: "{title}"{year and f", Year: {year}" or ""}')
     results = search(title, [required_type], year)
+    if (results == [] and year is not None): # no match, possibly a mismatch in year metadata
+        msg = f'Trakt search yielded no results for the {required_type}, {title}, Year: {year}. Retrying search without filtering by year'
```

Comment:
I'd prefer this line to be broken up - the line length should not exceed 88 characters. 
```suggestion
        msg = (f'Trakt search yielded no results for the {required_type}, {title}, '
               f'Year: {year}. Retrying search without filtering by year.')
```